### DISCUSSION TITLE
wslu 3.2.3 LTS

### DISCUSSION THREAD

**Discussion by car-meows** [wslutilities/wslu#186] [General]
## :bug: Bug Fixes

- wslsys: fix an issue in `call_windows_uptime` (fixed by #182, manually merged)
- header: fix a distro detection issue in Ubuntu Impish (21.10)

<hr /><em>This discussion was created from the release <a href='https://github.com/wslutilities/wslu/releases/tag/v3.2.3'>wslu 3.2.3 LTS</a>.</em>

**Comment by tranzystorekk** [comment]
Seems like the `.zst` package hasn't been created for this release 🙁

**Reply by car-meows** [reply]
Sorry for that 😅 It is currently uploaded: https://github.com/wslutilities/wslu/releases/download/v3.2.3/wslu-3.2.3-0-any.pkg.tar.zst

**Reply by tranzystorekk** [reply]
Thanks a lot! ❤️
